src/tests/test_rest_framework_dso/test_views.py

```diff
@@ -632,3 +632,24 @@ def test_list_count_excluded(self, api_client, django_assert_num_queries, data):
         assert "X-Pagination-Count" not in response
         assert "totalElements" not in data["page"]
         assert "totalPages" not in data["page"]
+
+    # Test that all object are returned during pagination
+    @pytest.mark.parametrize("page_size_param", ["_pageSize", "page_size"])
+    @pytest.mark.parametrize("movie_objects", [{"foo", "bar", "baz", "xyz"}])
```

Comment:
Omdat `movie_objects` een lijst is die maar 1 element bevat, kun je die beter gewoon binnen de body van de testfunctie definieren. De `parametrize` is alleen interessant als je over meerdere parameters wilt loopen.

Reply:
Fixed 👍